spec/services/search_service_spec.rb

```diff
@@ -247,21 +239,21 @@ def commodity_pattern(item)
       end
 
       context 'when non declarable' do
-        let!(:heading) do
+        before do
           create :heading, goods_nomenclature_item_id: '8418000000',
                            validity_start_date: Date.new(2011, 1, 1)
-        end
-        let!(:commodity1) do
+
           create :commodity, :with_indent,
-                 indents: 3,
-                 goods_nomenclature_item_id: '8418213100',
+                 indents: 4,
```

Comment:
`parent: commodity` ? Could possibly drop manual assignment of item_id on both commodities as well as well?